### DISCUSSION TITLE
conflict with hen stores

### DISCUSSION THREAD

**Discussion by eddbull** [aldostools/webMAN-MOD#1100] [General]
Good evening, I updated my ps3 hfw to 4.91, and now the webmam mod is conflicting with my "zukostore" and "jymsstore" stores, the webman mod is not showing up in the games folder, I already reinstalled hen and nothing, It only resolves if I uninstall the stores, could you resolve this?  I'll be very thankfull

**Comment by aldostools** [comment]
It is not an issue of webMAN MOD... ask to the developers of these pirate stores that resolve the issue that they caused.

**Reply by eddbull** [reply]
all good

**Comment by eddbull** [comment]
I found out why the webman mod folder isn't showing up after installing the stores, it's because the stores are preventing xmb from reloading

**Reply by aldostools** [reply]
Excellent!

**Comment by OsamaXJ** [comment]
Same thing happened to me, and resolved by deleting stores folder from hdd0/game, then reinstalled the hfw through safe mode.

**Comment by aldostools** [comment]
These "stores" probably are replacing `category_game.xml` in dev_flash with one that does not contain the entry for `fb.xml` needed by webMAN MOD to show webMAN Games menu.

That is not an issue of webMAN MOD. These stores are created by noobs that ignore how webMAN MOD works.
